install/boards/bcm_27xx.sh

```diff
@@ -34,9 +34,9 @@ for STRING in \
     "dtoverlay=uart4" \
     "dtoverlay=uart5" \
     "dtparam=i2c_vc=on" \
-    "dtoverlay=i2c1" \
-    "dtoverlay=i2c4,pins_6_7" \
-    "dtoverlay=i2c6,pins_22_23" \
+    "dtoverlay=i2c1,baudrate=1000000" \
+    "dtoverlay=i2c4,pins_6_7,baudrate=1000000" \
+    "dtoverlay=i2c6,pins_22_23,baudrate=400000" \
```

Comment:
the clock can be configured by the driver interface, is there a particular reason to add it here ?

Reply:
Do you mean the driver in ardupilot? it looks like it isn't implemented there. `bus_clock` is not consumed:
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_Linux/I2CDevice.cpp#L346